src/controllers/user.controllers.ts

```diff
@@ -120,6 +124,84 @@ export const userLogin = async (req: Request, res: Response) => {
   }
 };
 
+export const sendResetInstructions = async (
+  req: Request,
+  res: Response,
+  next: NextFunction
+) => {
+  try {
+    const foundUser = await User.findOne({
+      where: {
+        email: req.body.email
+      }
+    });
+    if (!foundUser) return res.status(404).json({ msg: "User not found" });
+    const { firstName, id, email } = foundUser.dataValues;
+
+    const token = jwt.sign(
+      {
```

Comment:
We have a function that does this operation, use it.

Reply:
Fixed it